src/simplicial-hott/08-covariant.rzk.md

```diff
@@ -814,8 +814,82 @@ is covariant as shown above. Transport of an `e : C x` along an arrow
     ( center-contraction (dhom-from A x y f C u) (is-covariant-C x y f u))
     ( lift)
     ( homotopy-contraction (dhom-from A x y f C u) (is-covariant-C x y f u) lift)
+
+#def covariant-uniqueness-curried
+  ( A : U)
+  ( x y : A)
+  ( f : hom A x y)
+  ( C : A → U)
+  ( is-covariant-C : is-covariant A C)
+  ( u : C x)
+  : ( v : C y)
+  -> ( dhom A x y f C u v)
+  -> ( covariant-transport A x y f C is-covariant-C u) = v
+  :=
+    \ v g -> covariant-uniqueness A x y f C is-covariant-C u (v, g)
+
```

Comment:
The empty lines and the beginning and the end of a block are probably unnecessary, since are rendered into just extra space. But I do not feel strongly about this.